backend/src/index.ts

```diff
@@ -61,8 +62,24 @@ async function main() {
   // Express Server
   const server = express()
 
+  const plugins = [
+    {
+      requestDidStart() {
+        return {
+          willSendResponse(requestContext: any) {
+            const { setHeaders = [] } = requestContext.context
+            setHeaders.forEach(({ key, value }: { [key: string]: string }) => {
+              requestContext.response.http.headers.append(key, value)
+            })
+            return requestContext
```

Comment:
I couldn't determine where you send the header with this? mind explaining?

Reply:
This is the only way I got this working. I have an array `setHeaders` in the context. To this array I push the token for the response header. Then, when `willSendResponse` is triggered I loop over the array and write the response headers. The example you gave only works for setting up a static header once.